myapp.rb

```diff
@@ -6,186 +6,50 @@
 require 'securerandom'
 require 'erb'
 require 'byebug'
+require 'pg'
 
-#postgreSQLで出力テスト
-conn = PG.connect( dbname: 'shinatra_memoapp' )
-# @memos = conn.exec( "SELECT * FROM memodata" ) #これ自体がハッシュなので、@memos[0]とかで取り出せそう
+include ERB::Util
+conn = PG.connect(dbname: 'shinatra_memoapp')
+conn.exec('CREATE TABLE IF NOT EXISTS memodata (id UUID PRIMARY KEY, title TEXT , sentence TEXT)')
 
-def load
-  JSON.parse(File.read('json/memo.json'), symbolize_names: true)
-end
-
-def store(memodata)
-  File.open('json/memo.json', 'w') do |file|
-   JSON.dump(memodata, file)
-  end
-end
-# 他に必要なもの
-  # メモリスト(トップページで一覧表示させるため)
-  # @memolist
-
-
-# top
 get '/memos' do
-  @memos = load()
-  @memo_id_array = @memos.keys
-  @array_size = @memo_id_array.size
-
+  @memos = conn.exec('SELECT * FROM memodata')
   erb :index
 end
 
-  # top
-  get '/memos' do #ここわからんかも
-    memos = conn.exec( "SELECT * FROM memodata" )
-    # 展開してhash作り直せば良い？
-      # 一旦無視 @memolist = @memo.map{ |memo| "<a href=/memos/#{memo[0]}>#{html_escape(memo[1][:title])}</a>" }.join("<br>")
-    erb :index
-  end
+get '/memos/new' do
+  erb :new
+end
 
 post '/memos' do
-  @memos = load()
-  @memo = { title: params[:memo_title].to_s, text: params[:memo_text].to_s }
-  @memos[SecureRandom.uuid] = @memo
-  store(@memos)
-
-  post '/memos' do
-    PostgreSQL.create
-    redirect '/memos'
-  end
+  id = SecureRandom.uuid
+  title = params[:memo_title]
+  text = params[:memo_text]
```

Comment:
```suggestion
  title = params[:title]
  text = params[:text]
```

パラメータ名が少し冗長なので`memo_`はカットしてもいいかも。
更新時も同様です。

Reply:
ありがとうございます。
f4f7568 が該当commitになります。
よろしくお願いいたします。